src/main/java/com/example/bookshop/exception/CustomGlobalExceptionHandler.java

```diff
@@ -41,6 +41,11 @@ public ResponseEntity<String> handleNullPointerExceptions(EntityNotFoundExceptio
         return new ResponseEntity<>("Entity not found", HttpStatus.NOT_FOUND);
     }
 
+    @ExceptionHandler(RegistrationException.class)
+    public ResponseEntity<String> handleNullPointerExceptions(RuntimeException ex) {
```

Comment:
```suggestion
    public ResponseEntity<String> handleRegistrationExceptions(RegistrationException ex) {
```
be attentive with copy-paste. Check other methods